src/main/java/org/folio/linked/data/service/validation/validators/LccnValidator.java

```diff
@@ -0,0 +1,56 @@
+package org.folio.linked.data.service.validation.validators;
+
+import java.util.regex.Pattern;
+import org.folio.linked.data.domain.dto.InstanceField;
+import org.folio.linked.data.domain.dto.LccnField;
+import org.folio.linked.data.domain.dto.LccnRequest;
+import org.folio.linked.data.domain.dto.ResourceRequestDto;
+import org.folio.linked.data.exception.ValidationException;
+import org.folio.linked.data.service.validation.LdValidator;
+import org.folio.linked.data.service.validation.ValidationContext;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component
+@Qualifier(LccnValidator.CODE)
+public class LccnValidator implements LdValidator<ResourceRequestDto> {
```

Comment:
name looks a bit generic, we have other validations to be added soon, e.g. lccn uniqueness, so i'd suggest to rename it for example to LccnStructureValidator

Reply:
done